packages/db/src/test/nameRecord.graphql.ts

```diff
@@ -31,26 +31,11 @@ export const GetAllNameRecords = gql`
 `;
 
 export const AddNameRecord = gql`
-  input ResourceInput {
-    id: ID!
-  }
-
-  input PreviousNameRecordInput {
-    id: ID!
-  }
-
-  input NameRecordInput {
-    name: String!
-    type: String!
-    resource: ResourceInput!
-    previous: PreviousNameRecordInput
-  }
-
   mutation AddNameRecord(
     $name: String!
     $type: String!
-    $resource: ResourceInput!
-    $previous: PreviousNameRecordInput
+    $resource: ResourceReferenceInput!
```

Comment:
Maybe I'm just missing it but where is this type imported?

Reply:
No need to import these references, since queries get all the identifiers defined by the schema. So the schema defines `ResourceReferenceInput`, thus making it available to anyone who wants to query Truffle DB, without the need to define something like that themselves. You'll notice that's how I got away with deleting all those various `input` definitions everywhere.